src/py/flwr/server/app.py

```diff
@@ -374,13 +375,18 @@ def run_superlink() -> None:
                 server_private_key,
                 server_public_key,
             ) = maybe_keys
-            interceptors = [
-                AuthenticateServerInterceptor(
-                    client_public_keys,
-                    server_private_key,
-                    server_public_key,
-                )
-            ]
+            state = state_factory.state()
+            state.store_client_public_keys(client_public_keys)
+            state.store_server_public_private_key(
+                public_key_to_bytes(server_public_key),
+                private_key_to_bytes(server_private_key),
+            )
```

Comment:
Should we change the public-private oder to private-public to align with the rest of the codebase?